src/lib/subtract.js

```diff
@@ -1,4 +1,6 @@
-const subtract = (predicate, array) => {
-
+const subtract = (num1,num2) => {
```

Comment:
This could be made shorter:
` const subtract = ( num1, num2 ) => num1 - num2 `

When writing in ES6, and using arrow functions, if your function's logic is short enough to fit on one line, you may omit curly braces and there is an implicit return, so you don't even need to type out `return num1 - num2`